internal/clients/diffreviewer/circleci/circleci_service.go

```diff
@@ -199,8 +199,19 @@ func (s *Service) WriteComments(comments []*diffreviewer.Comment) error {
 		return nil
 	}
 	if s.PrDetails.PrNumber != nil {
+		existingComments, _, err := s.GithubClient.PullRequestsService().ListComments(
```

Comment:
Is there a way to filter the comments we get? I assume this would also fetch the comments from engineers reviewing the diff

Reply:
Yea good question. It doesn't look like we can pass in any relevant filters in the `PullRequestListCommentsOptions`, as those filters are just `sort` (created vs updated comment), `direction` (asc vs desc), and `since` (time).

Since the comment is written with a user's personal access token, the comment will appear the same as a user-written one. What I could do is iterate through all the comments we get back, and only retain the one's where the comment body start with `Suspicious content detected`, but I'm not sure if that actually addresses your question (or even cuts down the runtime).